### DISCUSSION TITLE
Show index of items from related module

### DISCUSSION THREAD

**Discussion by rochmirabeau** [YetiForceCompany/YetiForceCRM#15352] [How to] [answered]
I have a module that has a one to many relation with another, I'm showing the related module at the top of the summary view.  Is there a way to show the index/number for each related item (i.e. 1/3, 2/3, 3/3)?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
where?  attach a screenshot

**Reply by rochmirabeau** [reply]
https://imgur.com/a/mg4XTbB

**Reply by mariuszkrzaczkowski** [reply]
layouts\basic\modules\Vtiger\RelatedListContents.tpl
![image](https://user-images.githubusercontent.com/10198654/138510296-909cc481-0517-4f63-9c8d-bf651a553930.png)
![image](https://user-images.githubusercontent.com/10198654/138510105-e846dd1b-034a-42bf-9e2c-5ae3e8bb0bf2.png)

**Reply by rochmirabeau** [reply]
Boom! Thank you!
